component/cart/cart.html

```diff
@@ -0,0 +1,159 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://webkit-ui.netlify.app/style.css">
+    <link rel="stylesheet" href="/css/style.css">
+    <script src="https://kit.fontawesome.com/85de537a36.js" crossorigin="anonymous"></script>
+    <title>Cart</title>
+</head>
+
+<body>
+    <div class="h-100">
+
+
+        <!--- Navigation bar -->
+        <header class=" bg-green-7 text-color-0 pd-5 position-fixed t-0">
+            <div class="wt-80  m-auto  flex items-center">
+                <div class="text-xm">
+                    <a href="/index.html" target="_blank" class="text-dec text-color-0 text-m">Farmgistic</a>
+                </div>
+                <div class="flex items-center justify-even m-lf">
+                    <input type="search" placeholder="search"
+                        class="pd-y-3 pd-x-5 rounded-L border-none text-center outline-none">
+                    <i class="fa-solid fa-magnifying-glass bg-black-0 text-color-9 rounded-L pd-3 text-xm"></i>
+                </div>
+                <div class="m-lf">
+
+
+                    <ul class="style-none flex items-center">
+                        <li class="pd-x-4">
+                            <a href="/component/login/login.html" target="_blank"
+                                class="text-dec text-color-0 pd-x-5 pd-y-3 bg-red-7">Login</a>
```

Comment:
Generally cart is visible only after logging in, if possible you can remove the login button

Reply:
Ya sure Srikar.